Sources/HipMobileUI/Views/AchievementsScreenView.xaml

```diff
@@ -69,8 +69,10 @@
             </StackLayout>
             <ContentView HorizontalOptions="Fill" VerticalOptions="FillAndExpand" />
             <StackLayout HorizontalOptions="FillAndExpand" VerticalOptions="End" Orientation="Horizontal"
-                         BackgroundColor="{StaticResource PrimaryColor}" Padding="20">
+                         BackgroundColor="{StaticResource PrimaryColor}" IsVisible="{Binding IsLoggedIn}"
+					   	 Padding="20">
                 <Label FontSize="24" Text="{Binding Score}" FontAttributes="Bold"
+					   IsVisible="{Binding IsLoggedIn}"	
```

Comment:
Setting the property _IsVisible_ of this label is redundant since it's a control within the structure of the _StackLayout_. Hiding parent also hides its children. You can safely remove it.